src/components/MainPage.js

```diff
@@ -23,19 +24,29 @@ const styles = () => ({
     width: 445,
   },
   card: {
-    maxWidth: 445,
+    maxWidth: 345,
+    minHeight: 600,
     margin: '20px auto',
   },
   media: {
     height: 0,
     paddingTop: '100%',
   },
+  container: {
+    padding: 0,
+  },
 });
 
+type Action = { type: string, payload: Object };

```

Comment:
I think that this will be reused across the project, so maybe better will be placing this types to some shared folder?